src/pyocr/tesseract.py

```diff
@@ -161,6 +161,15 @@ def can_detect_orientation():
     )
 
 
+def psm_parameter():
+    """Return the psm option string depending on the Tesseract version."""
+    version = get_version()
+    if version[0] <= 3:
```

Comment:
You could have used Python ternary operator: `return "--psm" if version[0] > 3 else "-psm"`

Reply:
Fixed this in the updated pull request.